src/components/MagicFormFiller.tsx

```diff
@@ -0,0 +1,140 @@
+import { sleep } from '@bundlr-network/client/build/common/utils';
+import { Button, Flex, Text } from '@chakra-ui/react';
+import { useCallback, useState } from 'react';
+import {
+  goToStep,
+  RecipientSetByOption,
+  selectArchaeologist,
+  setCustomResurrectionDate,
+  setFile,
+  setName,
+  setRecipientState,
+  setResurrection,
+  setResurrectionRadioValue,
+} from 'store/embalm/actions';
+import { Step } from 'store/embalm/reducer';
+import { useDispatch, useSelector } from 'store/index';
+
+// The delay between actions in ms. Set to 0 for the form to fill instantaneously.
+const delay = 500;
```

Comment:
what is the usecase for this (+ the sleeps)? seems like we'd always want it to auto-fill state and skip to the last step. But this is super easy to update so no issue merging it, was just curious.